### PR TITLE
added welcome workflow

### PR DESCRIPTION
**Pull request by SaumyaBhushan**
### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [ ] Search for duplicates.
- [ ] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by SaumyaBhushan** [comment]
Hi, @crescentpartha I have added this workflow to this project because there was no workflow. Using this, an automatic welcome message will be triggered by the git bot for new users.

**Comment by SaumyaBhushan** [comment]
Are you willing to merge it in your project

**Comment by crescentpartha** [comment]
Can you explain this workflow elaborately? @SaumyaBhushan 
Or make some changes so that it will give a welcome message only for those who `create a pull request`. 
I am using this from `windows`, you set up this in Linux-latest, will it work?

File: .github/workflows/welcome.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  push:
+    branches: [ main ]
+  issues:
+    types: [ opened ]
+  issue_comment:
+    types: [ created ]
+  pull_request_target:
+    types: [ opened ]
+  pull_request_review_comment:
+    types: [ created ]
```
Comment (crescentpartha) [inline .github/workflows/welcome.yml:12]:
```suggestion
  issues:
    types: [ opened ]
  pull_request_target:
    types: [ opened ]
```

**Review by crescentpartha — CHANGES_REQUESTED** [review]

**Comment by SaumyaBhushan** [comment]
@crescentpartha It will work for Linux and I can modify it only for pull request let me commit those changes

**Comment by SaumyaBhushan** [comment]
@crescentpartha Github actions bot will print message this message ==> It''s great having you contribute to this project</h1> Welcome to the community :nerd_face: :rocket:'
        It''s great having you contribute to this project.  Welcome to the community :nerd_face:'
         'Thanks for joining our community - we help and encourage each other to contribute to open source little and often 🤓 . Any questions let us know.'

**Review by crescentpartha — APPROVED** [review]
Approved changes.

**Pull request merged**